### ISSUE TITLE
[UI] Convert `MesherySettingsPerformanceComponent` class-based component to functional component

### ISSUE THREAD

**Issue by Rajdip019** [meshery/meshery#8813]
### Current Behavior
`MesherySettingsPerformanceComponent` is a  class-based component

### Desired Behavior
`MesherySettingsPerformanceComponent` should be a functional component

### Implementation
Convert all the Class-based components to new updated functional components. It is important here to convert a class-based component to a functional component you may need to convert some functions as well. Check if the function is used in other components or not.

Also, make sure to use the hooks while converting them. For example, you need to covert withNotify with useNotification, useRouter hook. Please check if a hook for functional components is available and use that.

Here is a quick blog that will help you get started with : [Link](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by csengineer1990** [comment]
hi @Rajdip019 I am a newcomer.  can I work on this sir?

**Comment by Rajdip019** [comment]
Sure. Do you have local up and running?

**Comment by csengineer1990** [comment]
yes sir yesterday I set it up, and just one thing left when I run locally not get the data.

**Comment by Rajdip019** [comment]
Follow this : https://docs.meshery.io/project/contributing/contributing-ui

**Comment by csengineer1990** [comment]
ok sir. Yes sir i am working on that. when run make ui-setup and make ui-server, make server works fine.

**Comment by Rajdip019** [comment]
Then do make-ui and make-server in different consoles.

**Comment by EraKin575** [comment]
> ok sir. Yes sir i am working on that. when run make ui-setup and make ui-server, make server works fine.

you can also just run `make ui-server` . When you change the ui, just run `make ui-build while the server is running`

**Comment by csengineer1990** [comment]
ok sir @Rajdip019. 
ok sir@EraKin575

**Comment by ishwar170695** [comment]
`ui/components/settings/MesherySettingsPerformanceComponent.tsx` is now implemented as a functional component using React hooks. The requested migration has been completed, so closing the issue.
